packages/sdk/src/client.ts

```diff
@@ -782,28 +782,14 @@ export class Client
             }),
         )
 
-        const response = await this.rpcClient.createStream({
+        const response = await this.rpcClient.createMediaStream({
             events: [inceptionEvent],
             streamId: streamIdAsBytes(streamId),
         })
 
-        const unpackedResponse = await unpackStream(response.stream, this.unpackEnvelopeOpts)
-        const streamView = new StreamStateView(this.userId, streamId)
-        streamView.initialize(
-            unpackedResponse.streamAndCookie.nextSyncCookie,
-            unpackedResponse.streamAndCookie.events,
-            unpackedResponse.snapshot,
-            unpackedResponse.streamAndCookie.miniblocks,
-            [],
-            unpackedResponse.prevSnapshotMiniblockNum,
-            undefined,
-            [],
-            undefined,
-        )
+        check(response?.nextCreationCookie !== undefined, 'nextCreationCookie must be defined')
```

Comment:
A "check(...)" is basically an assert. My normal rule is to not assert on data/objects that come from external sources (in this case the response from a remote network call). 

In this case... we're expecting this call to either throw and error or have proper data right? So I guess this is fine. Can you update the error message to be more explicit? Something like `nextCreationCookie was expected but was not returned in response?`